src/projections/airocean.cpp

```diff
@@ -0,0 +1,251 @@
+/* enable predefined math constants M_* for MS Visual Studio */
+#if defined(_MSC_VER) || defined(_WIN32)
+#ifndef _USE_MATH_DEFINES
+#define _USE_MATH_DEFINES
+#endif
+#endif
+
+#include <cmath>
+#include <cstdint>
+#include <errno.h>
+
+#include "proj.h"
+#include "proj_internal.h"
+
+PROJ_HEAD(airocean, "Airocean") "\n\tMisc, Sph&Ell";
```

Comment:
The docs in this PR specify that only spherical form is supported, so probably remove `&Ell` unless it is mistaken.